btc-trigger/nodejs/src/index.ts

```diff
@@ -0,0 +1,258 @@
+import { LitNodeClient } from "@lit-protocol/lit-node-client";
+import { LitNetwork, LIT_RPC } from "@lit-protocol/constants";
+import {
+  createSiweMessageWithRecaps,
+  generateAuthSig,
+  LitAbility,
+  LitActionResource,
+  LitPKPResource,
+} from "@lit-protocol/auth-helpers";
+import { LitContracts } from "@lit-protocol/contracts-sdk";
+import BN from "bn.js";
+import mempoolJS from "@mempool/mempool.js";
+import fetch from "node-fetch";
+import elliptic from "elliptic";
+import * as bitcoin from "bitcoinjs-lib";
+import * as ethers from "ethers";
+import * as ecc from "tiny-secp256k1";
+import * as bip66 from "bip66";
+import * as crypto from "crypto";
+
+import { litActionCode } from "./litAction";
+
+const PKP_PUBLIC_KEY = process.env["PKP_PUBLIC_KEY"]!;
+const ETHEREUM_PRIVATE_KEY = process.env["ETHEREUM_PRIVATE_KEY"]!;
+const BTC_DESTINATION_ADDRESS = process.env["BTC_DESTINATION_ADDRESS"]!;
+const LIT_CAPACITY_CREDIT_TOKEN_ID =
+  process.env["LIT_CAPACITY_CREDIT_TOKEN_ID"];
```

Comment:
Which of these are required? We should be using the `getEnv` util function for the required ones